Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Remediation_Path_Rules_README.md

```diff
@@ -1,27 +1,23 @@
-This playbook returns "RemediationAction" options based on the return from the Remediation Path Rules API, or defaults to the data collection task.
+This playbook returns "RemediationAction" options based on the return from the Remediation Path Rules API, or defaults to data collection task.
 
 ## Dependencies
 
 This playbook uses the following sub-playbooks, integrations, and scripts.
 
 ### Sub-playbooks
 
-Cortex ASM - Remediation Objectives
+* Cortex ASM - Remediation Objectives
```

Comment:
```suggestion
Cortex ASM - Remediation Objectives
```

Reply:
@ShirleyDenkberg , any reason why you get rid of this `*`? it is added by the SDK